DataStreamerUWP/Service/DataStreamerAppService.cs

```diff
@@ -76,16 +77,23 @@ private async void OnRequestReceived(AppServiceConnection sender, AppServiceRequ
 
                         // Handle events/notifications from Data Streamer  
                         case "Event":
-                            await _service.OnEvent(message);
+                            // Run asynchronously
+                            _service.OnEvent(message).FireAndForget();
                             break;
 
                         case "Close":                       
-                            await _service.Disconnect();
+                            // Run asynchronously
+                            _service.Disconnect().FireAndForget();
                             break;
 
                         default:
                         {
-                            returnData.Add("Status", "Fail: unknown command");
+                            var jsonParams = message.ValueOrDefault("Params", "").ToString();
+                            var parms = jsonParams == null ? null : JObject.Parse(jsonParams);
+
```

Comment:
unwanted newline

Reply:
Don't see what you're talking about but I did remove the commented out line.